packages/boa/esm/loader.mjs

```diff
@@ -0,0 +1,40 @@
+import boa from '../lib/index.js';
+
+const extensionsPrefix = 'py:';
+const passBaseUrl = 'file://py/';
+
+export function resolve(specifier, context, defaultResolve) {
+  if (specifier.startsWith(extensionsPrefix)) {
+    return {
+      url: passBaseUrl + specifier.replace(extensionsPrefix, '')
+    };
+  }
+  return defaultResolve(specifier, context, defaultResolve);
+}
+
+export function getFormat(url) {
+  let format;
+  if (url.startsWith('nodejs:')) {
+    format = 'builtin';
+  } else if (url.startsWith(passBaseUrl)) {
+    format = 'dynamic';
+  }
+
+  return {
+    format: format || 'module'
+  };
+}
+
+export function dynamicInstantiate(url) {
+  const moduleInstance = boa.import(url.replace(passBaseUrl, ''));
+  const moduleExports = [ 'default', ...Object.keys(moduleInstance) ];
+  return {
+    exports: moduleExports,
```

Comment:
Need to pass named, can't think of a way to get py module's properties. @yorkie Any good suggestions？

Reply:
Boa fetches the property object dynamically, can esm loader support absolute dynamic way?

Reply:
As for fetching the props statically, you can try with `obj.__dict__`, which returns the properties of the given object.

Reply:
fixed